jmp-service-rest/src/main/java/mzaba/subscription/rest/service/controller/ServiceController.java

```diff
@@ -0,0 +1,21 @@
+package mzaba.subscription.rest.service.controller;
+
+import mzaba.subscription.rest.domain.dto.SubscriptionRequestDto;
+import mzaba.subscription.rest.domain.dto.SubscriptionResponseDto;
+import org.springframework.http.ResponseEntity;
+
+import java.util.List;
+
+
+public interface ServiceController {
```

Comment:
Usually, we add **Api** prefix when we define API (controller without implementation). So `ServiceApi`, `ControllerApi `would be better. Additionally, you can move `@GetMapping`, `@PostMapping` annotations to these interfaces because here we define our api contract